rwengine/src/loaders/LoaderIDE.cpp

```diff
@@ -65,8 +65,7 @@ bool LoaderIDE::load(const std::string &filename, const PedStatsList &stats) {
                     break;
                 case OBJS:
                 case TOBJ: {  // Supports Type 1, 2 and 3
-                    auto objs =
-                        std::unique_ptr<SimpleModelInfo>(new SimpleModelInfo);
+                    std::unique_ptr<SimpleModelInfo> objs = std::make_unique<SimpleModelInfo>();
```

Comment:
In this switch case nothing happens with the result obj... can't see why it needs to be on the heap?